packages/firestore/test/integration/util/helpers.ts

```diff
@@ -192,7 +192,7 @@ export async function withTestDbsSettings(
   }
 
   try {
-    await fn(dbs);
+    return await fn(dbs);
```

Comment:
If we return here, can it go to the `finally` block?

Reply:
Yes, it will still execute the `finally` block.